packages/@aws-cdk/aws-glue-alpha/README.md

```diff
@@ -24,19 +24,6 @@ service that makes it easier to discover, prepare, move, and integrate data
 from multiple sources for analytics, machine learning (ML), and application
 development.
 
-Without an L2 construct, developers define Glue data sources, connections,
-jobs, and workflows for their data and ETL solutions via the AWS console,
-the AWS CLI, and Infrastructure as Code tools like CloudFormation and the
-CDK. However, there are several challenges to defining Glue resources at
-scale that an L2 construct can resolve. First, developers must reference
-documentation to determine the valid combinations of job type, Glue version,
-worker type, language versions, and other parameters that are required for specific
-job types. Additionally, developers must already know or look up the
-networking constraints for data source connections, and there is ambiguity
-around how to securely store secrets for JDBC connections. Finally,
-developers want prescriptive guidance via best practice defaults for
-throughput parameters like number of workers and batching.
```

Comment:
this simply is not consistent with the voice of READMEs in the rest of the modules. we do not reference anything beyond how the current L2 constructs can be used. the rest of the README needs work, but this particular paragraph isn't necessary